versioneer.py

```diff
@@ -2099,7 +2170,7 @@ def make_release_tree(self, base_dir, files):
 """
 
 
-def do_setup():
+def do_setup() -> int:
     """Do main VCS-independent setup function for installing Versioneer."""
```

Comment:
One-line sphinx docstring (missing the description of input parameters, errors raised, what's returned, etc.: :param :return)